ch3/exercise3-10/comma.go

```diff
@@ -27,9 +27,9 @@ func comma(s string) string {
 	}
 	s = s[offset:]
 	
-	for i := len(s); i > 0; i -= 3{
-		buf.WriteString(s[i-3 : i])
-		if i > 3 {
+	for i := 0; i < len(s); i += 3{
```

Comment:
Alternative, no need to re-slice, and start at `i := offset`. Doesn't matter one way or the other, just an idle thought.